uat/testing-features/src/main/java/com/aws/greengrass/LocalDeploymentSteps.java

```diff
@@ -48,6 +48,7 @@
 public class LocalDeploymentSteps {
```

Comment:
This will be removed from the shadow manager repo once the code is moved to OTF repo.